src/Components/CreateImageWizard/formComponents/FileSystemConfiguration.js

```diff
@@ -100,7 +100,7 @@ const FileSystemConfiguration = ({ ...props }) => {
       setItemOrder(newRows.map((row) => row.id));
       change('file-system-config-radio', 'manual');
     }
-  }, [customizations, isSuccess]);
+  }, [customizations, isSuccess, hasCustomizations, change, rows]);
```

Comment:
Use this pattern instead:
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
as we discussed in the group channel, we agreed to add those dependencies
and in the future maybe we will refector the code and remove the useEffect hook were ever we can